darts/models/forecasting/regression_model.py

```diff
@@ -314,6 +314,11 @@ def extreme_lags(
             max_future_cov_lag,
         )
 
+    @property
+    def supports_multivariate(self) -> bool:
+    """Multivariate support is obtained by wrapping the univariate model in a `sklearn.multioutput.MultiOutputRegressor`"""
```

Comment:
let's rephrase a little, to cover multivariate support for all regression models
```suggestion
        """
        If available, uses `model`'s native multivariate support. If not available, obtains multivariate support by
        wrapping the univariate model in a `sklearn.multioutput.MultiOutputRegressor`.
        """
```